### PR TITLE
DDI 406 [Contextual Doc] Plan your set up

### PR DESCRIPTION
**Pull request by Mercy811**
# Amplitude Developer Docs PR

## Description

preview: https://pr-599.d19s7xzcva2mw3.amplifyapp.com/analytics/plan-your-set-up/

This is the second step of our new onboarding process after What is Amplitude page which gives new users a high level overview of what Amplitude is. This page lists out high level steps on how to get started, necessary technical considerations to make, and what events to track upfront.

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [x] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [x] I previewed my documentation on a local server using `mkdocs serve`.
- [x] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/analytics/plan-your-set-up.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Amplitude recommends to get started with two important events in your product to help you get initial insights. 
+
+Here are some sample questions to get you thinking:
+
+|Questions|Events|
+|---------------|---------------|
+|How many daily active users or how many logins per week do I get? | **Login** event|
+|What percentage of users who add an item to their cart successfully checkout? | **Add to Cart** event and **Checkout** event|
+|What percentage of sign ups request a demo? | **Sign Up** event and Request Demo event|
+|What's the retention rate? How many users come back to the product after signing up 2 weeks later? | **Sign Up** event and **Session Start** event|
```
Comment (haoliu-amp) [inline docs/analytics/plan-your-set-up.md:48]: Why is retention related to Session Start event?

File: docs/analytics/plan-your-set-up.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Amplitude recommends to get started with two important events in your product to help you get initial insights. 
+
+Here are some sample questions to get you thinking:
+
+|Questions|Events|
+|---------------|---------------|
+|How many daily active users or how many logins per week do I get? | **Login** event|
+|What percentage of users who add an item to their cart successfully checkout? | **Add to Cart** event and **Checkout** event|
+|What percentage of sign ups request a demo? | **Sign Up** event and Request Demo event|
```
Comment (haoliu-amp) [inline docs/analytics/plan-your-set-up.md:47]: BOLD for "Request Demo"

**Review by haoliu-amp — CHANGES_REQUESTED** [review]

File: docs/analytics/plan-your-set-up.md (hunk 1 above)
Comment (Mercy811) [reply docs/analytics/plan-your-set-up.md:48]: IMO a Session Start event is fired when a user starts using the app. So we can get the retention rate of users coming back to the product after signing up 2 weeks later by 
```
number of sign-up users / number of users who come back after signing up 2 weeks later
= Sign up events / Session Start events after 2 weeks
```

File: docs/analytics/plan-your-set-up.md (hunk 2 above)
Comment (Mercy811) [reply docs/analytics/plan-your-set-up.md]: bolded it.

**Review by haoliu-amp — APPROVED** [review]

**Pull request merged**